unit_test/mock_util.cc

```diff
@@ -69,10 +69,17 @@ std::unique_ptr<fakeit::Mock<m::i_model>> get_mock_model(m::model_type_t model_t
 {
   auto mock = std::unique_ptr<Mock<m::i_model>>(new fakeit::Mock<m::i_model>());
 
-  const auto choose_rank_fn = [](const char*, uint64_t, r::string_view, std::vector<int>&, std::vector<float>&,
-                                  std::string& model_version, r::api_status*)
+  const auto choose_rank_fn = [](const char*, uint64_t, r::string_view, std::vector<int>& actions,
+                                  std::vector<float>& scores, std::string& model_version, r::api_status*)
   {
     model_version = "model_id";
+    actions.resize(2);
```

Comment:
some tests mock out the model rank call to this function
before the removal of explore_only the model would not get called and explore_only would populate the pdf

with that removed this function will get called when model rank is called, so mocking a pdf (no tests actually check what it is set to, only that is sums to 1)